backend/app/routes/template_routes.py

```diff
@@ -14,18 +14,17 @@ def getDraftTemplates():
 @app.route('/api/v1/template-form', methods=['GET'])
 # @token_required
 def getTemplate():
-    print("eeeeee")
     return TemplateFormController.getTemplateForm()
 
 
-@app.route('/api/v1/<user_id>/templates/', methods=['GET'])
-@token_required
+@app.route('/api/v1/templates/all', methods=['GET'])
+# @token_required
 def getAllTemplates():
+    print("hersdsdsde")
```

Comment:
Remove these reference print statements.

Reply:
Sorry, I will clear all the debugging statements